src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1164,3 +1164,7 @@ Glyphicon for success status messages
     padding-right: 2px;
     content: '\e084';
 }
+
```

Comment:
@whipermr5 For some reason, I'm having a hard time figuring what to say in the comment 😅 

As of now I've just come up with `Alert color for visibility-title`. Is that a good enough explanation? Do you have something better in mind?